packages/flutter/lib/src/foundation/memory_allocations.dart

```diff
@@ -0,0 +1,176 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' as ui;
+
+///
+abstract class ObjectEvent{}
+
+///
+typedef ObjectEventListener = void Function(ObjectEvent);
+
+///
+class ObjectCreated implements ObjectEvent {
+  ///
+  ObjectCreated({
+    required this.library,
+    required this.klass,
```

Comment:
Style guide says don't do this: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#spell-words-in-identifiers-and-comments-correctly